Header header
float32 top_lvl
float32 mid_lvl
float32 bot_lvl
